repour/adjust/project_manipulator_provider.py

```diff
@@ -17,6 +17,7 @@ def get_project_manipulator_provider(
     default_parameters,
     repour_parameters,
     rest_mode,
+    brew_pull_enabled,
```

Comment:
Instead of adding the param here I would remove it from https://github.com/project-ncl/repour/blob/master/repour/adjust/adjust.py#L447 because it cannot be used in the manipulator, because Indy does not support Brew pull for NPM packages. This would IMO bring more confusion into the code.

Reply:
Ahhh, definitely. I feel dumb. I blame this on lack of sleep!